examples/basic/variables.tf

```diff
@@ -33,3 +33,9 @@ variable "logdna_agent_tags" {
   description = "Array of string of tags for logdna agent"
   default     = []
 }
+
+variable "sysdig_agent_tags" {
```

Comment:
do not add this to the example - we will use the `resource_tags` variable for tagging all resources provisioned by an example

Reply:
Can you also remove `logdna_agent_tags` from the example - it should not be there